packages/Card/stories/Card.mdx

```diff
@@ -0,0 +1,200 @@
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import Card from "../src/";
+import DeluxeCard from "./examples/DeluxeCard.js";
+import Icon from "@paprika/icon/lib/Lock";
+import Avatar from "../src/components/Avatar";
+
+# Card
+
+The `<Card>` component is a customizable content container. It comes with subcomponents that allow for quick and easy creation of various card types for different use cases. Additionally any other child elements can be passed into the Card.
+
+
+Main features:
+
+- Ability to create custom cards.
+- Ability to use various preset sizes.
+
+## Installation
+
+```
+yarn add @/paprika/card
+```
+or
+```
+npm install @paprika/card
+```
+
+## Usage
+
+## Sub-Components
+
+The Card comes with several sub-components that can be sued to quickly create a desirable looking card. All sub-components have preset stylings. Alternatively users can choose to pass in their own elements. The following are a list of the available sub-components:
+
+
+- `<Card.Header/>`
+The header sub-component allows you to add a header section to the card.
+
+- `<Card.Content/>`
+The content sub-component allows you to add a container that can section of a part of the card.
+
+- `<Card.Title/>`
+The title sub-component allows you to add a title to the card.
+
+- `<Card.Separator/>`
+The separator sub-component allows you to add space to the card, you can adjust the size of the separator with its size property (small, medium, and large).
+
+- `<Card.Text/>`
+The text sub-component allows you to add a text body to the card.
+
+- `<Card.Metadata/>`
+The metadata sub-component allows you to add metadata to the card.
+
+- `<Card.Footer/>`
+The footer sub-component allows you to add a footer section to the card.
+
+### Small Card
+
+To create a small card simply add the size property to the `<Card/>` element and set it's value to small. You can then add several children elements of your choice.
+
+Note: The sizes are fixed therefore consideration will have to be given to size of content passed into the Card.
+
+```js
+<Card size="small">
+    <Card.Header>
+        <Avatar>
+        <Icon />
+        </Avatar>
+    </Card.Header>
+    <Card.Content>
+        <Card.Title>Card</Card.Title>
+        <Card.Metadata>100,000 total</Card.Metadata>
+    </Card.Content>
+</Card>
+```
+#### Result
+
+<Card size="small"><Card.Header><Avatar><Icon /></Avatar></Card.Header><Card.Content><Card.Title>Card</Card.Title><Card.Metadata>100,000 total</Card.Metadata></Card.Content></Card>
```

Comment:
<img width="343" alt="Screen Shot 2020-05-25 at 1 24 46 PM" src="https://user-images.githubusercontent.com/19940076/82841498-2bc20600-9e8b-11ea-9fd6-11e747448583.png">

The avatar is cut off a little here